ocean_lib/models/side_staking.py

```diff
@@ -78,3 +93,47 @@ def set_pool_swap_fee(
         return self.send_transaction(
             "setPoolSwapFee", (datatoken, pool_address, lp_swap_fee_amount), from_wallet
         )
+
+    def check_vesting_created_event(
```

Comment:
@trentmc what do you think about those functions that I use in `test_vesting.py`. Should I add them both as well at the bottom of `test_vesting.py` or keep them here? There were a lot of lines of code for retrieving vesting event.